jujugui/static/gui/src/app/components/deployment-flow/login/login.js

```diff
@@ -0,0 +1,246 @@
+              <div className="deployment-login__feature">
+                {this._generateTick()}
+                Hosted and managed juju controllers.
+              </div>
+              <div className="deployment-login__feature">
+                {this._generateTick()}
+                Reusable shareable models with unlimited users.
+              </div>
+            </div>
+          </div>
+          <div className="deployment-login__login">
+            <juju.components.USSOLoginLink
+              gisf={this.props.gisf}
+              callback={this.props.callback}
+              displayType="button"
+              loginToController={this.props.loginToController}>
+              Login
+            </juju.components.USSOLoginLink>
+          </div>
+          <div className="deployment-login__signup">
+            Do not have an account?
+            <juju.components.USSOLoginLink
+              gisf={this.props.gisf}
+              callback={this.props.callback}
+              displayType="text"
+              loginToController={this.props.loginToController}>
+              Sign up
+            </juju.components.USSOLoginLink>
+          </div>
+        </div>
+      </juju.components.DeploymentSection>);
+  }
+
+  /**
+   Generate the login for the direct deploy flow.
+
+   @return {Object} The login markup.
+  */
+  _generateDirectDeployLogin() {
+    const awsScale = 1.3;
+    const gceScale = 1.7;
+    const azureScale = 1;
+    return (
+      <div>
+        <juju.components.DeploymentSection
+          instance="deployment-login-signup">
+          <span className="deployment-login-signup__message">
+              Sign up to start deploying to your favourite cloud
+          </span>
+          <juju.components.USSOLoginLink
+            gisf={this.props.gisf}
+            callback={this.props.callback}
+            displayType="button"
+            loginToController={this.props.loginToController}>
+            Sign up
+          </juju.components.USSOLoginLink>
+          or&nbsp;
+          <juju.components.USSOLoginLink
+            gisf={this.props.gisf}
+            callback={this.props.callback}
+            displayType="text"
+            loginToController={this.props.loginToController}>
+            log in
+          </juju.components.USSOLoginLink>
+          to get started with&nbsp;
+          <a href="http://jujucharms.com/jaas">
+            JAAS
+          </a>
+        </juju.components.DeploymentSection>
+        <juju.components.DeploymentSection
+          instance="deployment-login-features">
+          <div className="six-col">
+            <h3>JAAS gives you Juju, as a service</h3>
+            <p>
+              JAAS is the best way to quickly model and deploy your cloud-based
+              applications. Concentrate on your software and the solutions with
+              a fully managed Juju infrastructure.
+            </p>
+            <p>
+              Learn more about&nbsp;
```

Comment:
This whole line should be the link and should be bolderised (actually just font-weight: normal).